### DISCUSSION TITLE
Praktische Nutzung von Datenbanken

### DISCUSSION THREAD

**Discussion by FlorianHeck2** [FlorianHeck2/codespaces-jupyter#8] [Announcements]
Diese Woche habe ich es nun endgültig implementiert, dass die Koordinaten eines Spielers im Spiel beim Schließen gespeichert werden. Wird das Spiel neu gestartet, so werden diese Koordinaten als Spielerkoordinaten gespeichert. Gewinnt man das Spiel (erreichen des Ziel), werden beim nächsten Start die Ursprungskoordinaten geladen. [(Zeile 14 und folgende)](https://github.com/FlorianHeck2/codespaces-jupyter/edit/main/project/main.py)

**Comment by 1Wargear** [comment]
Looks good, I would change the * int the SELECT to the columns in the right order so that if you change the table it doesn't break your whole code.

**Reply by FlorianHeck2** [reply]
nice hint, thanks

**Comment by LeoEnglert** [comment]
I find it cool that you using a database for storing this :) Maybe you could write a server for your game, so that we can play together in multiplayer :3 You could use your database for updating the position of each player.

**Reply by FlorianHeck2** [reply]
funny idea xD

**Comment by Rouge2123** [comment]
🥳 das ist zu cool! 
Hat es spaß gemacht das spiel so auszubauen?

**Reply by FlorianHeck2** [reply]
Spaß hat es schon gemacht. Dadurch, dass das Endprojekt davon unabhängig ist, hab ich aber leider vieles nicht umgesetzt, das ich als Idee hatte 😢
